### ISSUE TITLE
[macOS] AttributeError: 'CBCentralManager' object has no attribute 'isScanning'

### ISSUE THREAD

**Issue by mfilippo** [hbldh/bleak#234]
* bleak version: 0.6.4
* Python version: 3.7.0
* Operating System: macOS 10.12.6

### Description

I just wanted to run the basic scan:

```python
# scanner.py

import asyncio
from bleak import discover


async def run():
    devices = await discover()
    for d in devices:
        print(d)


loop = asyncio.get_event_loop()
loop.run_until_complete(run())
```
### What I Did

From my virtual environment:

```
$ python scanner.py
Traceback (most recent call last):
  File "scanner.py", line 12, in <module>
    loop.run_until_complete(run())
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 568, in run_until_complete
    return future.result()
  File "scanner.py", line 6, in run
    devices = await discover()
  File "/Users/matteo/virtualenvs/bt-scanner/lib/python3.7/site-packages/bleak/backends/corebluetooth/discovery.py", line 35, in discover
    await cbapp.central_manager_delegate.scanForPeripherals_(scan_options)
  File "/Users/matteo/virtualenvs/bt-scanner/lib/python3.7/site-packages/bleak/backends/corebluetooth/CentralManagerDelegate.py", line 121, in scanForPeripherals_
    while self.central_manager.isScanning():
AttributeError: 'CBCentralManager' object has no attribute 'isScanning'
```

**Comment by dlech** [comment]
`isScanning` was introduced in  macOS 10.13 so we will need a workaround for the call at https://github.com/hbldh/bleak/blob/release/v0.7.0/bleak/backends/corebluetooth/CentralManagerDelegate.py#L130 to support older macOS.

**Comment by hbldh** [comment]
The last minute fix in version 0.7.0 might just do the trick for this one. @mfilippo please try it out and report back here if a more rigorous fix is needed.

**Comment by mfilippo** [comment]
It's fixed. Thx!
